tests/DependencyInjection/Compiler/ExtensionCompilerPassTest.php

```diff
@@ -411,6 +417,12 @@ private function getContainer(): ContainerBuilder
             ->setClass(MockAdmin::class)
             ->setArguments(['', Article::class, CRUDController::class])
             ->addTag('sonata.admin');
+        $container
+            ->register('sonata_admin_without_entity')
+            ->setPublic(true)
+            ->setClass(MockAdmin::class)
+            ->setArguments(['', null, CRUDController::class])
```

Comment:
This lead to a deprecation
``` 
Passing other type than string as argument 2 for method Sonata\AdminBundle\Admin\AbstractAdmin::__construct() is deprecated since sonata-project/admin-bundle 3.65. It will accept only string in version 4.0.
```